game/src/test/java/com/hiddenswitch/spellsource/CustomCardsTests.java

```diff
@@ -2265,4 +2374,80 @@ public void testWyrmrestSniper() {
 			Assert.assertTrue(wyrmrest.hasAttribute(Attribute.STEALTH));
 		});
 	}
+
+	@Test
+	public void testAegwynn() {
+		List<String> spells = Arrays.asList("spell_fireball", "spell_arcane_explosion", "spell_flamestrike", "spell_frostbolt");
+		for (String spell : spells) {
+			runGym((context, player, opponent) -> {
+				shuffleToDeck(context, player, spell);
+				shuffleToDeck(context, player, "minion_aegwynn");
+				context.fireGameEvent(new GameStartEvent(context, player.getId()));
+				print("Spell Damage is " + player.getAttributeValue(Attribute.SPELL_DAMAGE) + " with " + spell);
+			});
+
+		}
+
+	}
+
+	@Test
+	public void testRelicRaider() {
+		runGym((context, player, opponent) -> {
+			shuffleToDeck(context, player, "weapon_vinecleaver");
+			playCard(context, player, "minion_relic_raider");
+			if (!player.getWeaponZone().isEmpty()) {
```

Comment:
Bit puzzling what's going on here. The weapon zone will never be empty, just assert it.